hypothesis-python/src/hypothesis/core.py

```diff
@@ -387,6 +387,24 @@ def skip_exceptions_to_reraise():
 exceptions_to_reraise = skip_exceptions_to_reraise()
 
 
+def failure_exceptions_to_catch():
+    """Return a tuple of exceptions meaning 'this test has failed', to catch.
+
+    This is intended to cover most common test runners; if you would
+    like another to be added please open an issue or pull request.
+    """
+    exceptions = [Exception]
+    try:  # pragma: no cover
+        from _pytest.outcomes import OutcomeException
+        exceptions.append(OutcomeException)
```

Comment:
Hmm. This looks too broad to me. This will also catch `Skipped`, which I don't think we want to catch - that will mean that we trigger shrinking etc. on skipped tests which if nothing else is needlessly slow.